scylladb/scylla-operator
# Update docs theme to 1.8

Related issue https://github.com/scylladb/sphinx-scylladb-theme/issues/1229

ScyllaDB Sphinx Theme 1.8 is out 🥳!  
![image](https://github.com/user-attachments/assets/64d8fc02-ea76-4128-b908-c79f8223b974)

Here's a summary of changes:
- Dark theme support for a fresh new look.
- Updated branding assets including new logos and mascots.
- Improved build process by recommending the inclusion of the `poetry.lock` file to ensure consistent dependencies in your CI/CD pipeline.
- New [tooltip component](https://sphinx-theme.scylladb.com/master/examples/tooltips.html) to add more context to glossary terms, configuration parameters, or any other elements without cluttering the page.

You can read more about all the notable changes [here](https://sphinx-theme.scylladb.com/master/upgrade/CHANGELOG.html#sept-2024).

## How to test this PR

1. Clone this PR.
2. Enter the docs folder, and run:
  
    ```
    make setup
    make preview
    ````

3. Open http://localhost:5500 in your browser. The doc should render without errors, and the version should be Sphinx Theme version (see the footer) must be ``1.8.x``:

    ![image](https://github.com/user-attachments/assets/d2c6cb4d-ec30-4459-8bea-84ed0eeb20fc)
